staging/src/k8s.io/client-go/plugin/pkg/client/auth/azure/azure.go

```diff
@@ -156,19 +158,22 @@ type tokenSource interface {
 }
 
 type azureTokenSource struct {
-	source    tokenSource
-	cache     *azureTokenCache
-	lock      sync.Mutex
-	cfg       map[string]string
-	persister restclient.AuthProviderConfigPersister
+	source             tokenSource
+	cache              *azureTokenCache
+	lock               sync.Mutex
+	omitTokenSPNPrefix bool
+	cfg                map[string]string
+	persister          restclient.AuthProviderConfigPersister
 }
 
 func newAzureTokenSource(source tokenSource, cache *azureTokenCache, cfg map[string]string, persister restclient.AuthProviderConfigPersister) tokenSource {
+	omitTokenSPNPrefix, _ := strconv.ParseBool(cfg[cfgOmitTokenSPNPrefix])
```

Comment:
I think we should handle the error here. If there is an error, we can log a warning message and default to `false` keeping it the way it is now?

Reply:
will pass in `bool`